<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1307</width>
    <height>1112</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEdit_f">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>0</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="enterF">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>0</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_v">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>50</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="enterV">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>50</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Select Line to View</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>-10</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Select File to Load</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="curLin">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>50</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>20</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Current Line</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>90</y>
      <width>1121</width>
      <height>821</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="Tab1">
     <attribute name="title">
      <string>Main View</string>
     </attribute>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="enterB">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>0</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Enter</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>321</width>
        <height>751</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_s">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>0</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="enterS">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>0</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Enter</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableView_S">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>30</y>
        <width>311</width>
        <height>321</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_t">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>350</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="enterT">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>350</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Enter</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableView_T">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>380</y>
        <width>311</width>
        <height>401</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="refresh">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>690</y>
        <width>381</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>22</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Refresh</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="selectE">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>50</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_2">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>160</y>
        <width>251</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Power Failure</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>370</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Track Heater Status</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>30</y>
        <width>281</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Select Errors</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>1</y>
        <width>381</width>
        <height>691</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="enterH">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>440</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Track Heater</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>10</y>
        <width>261</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>For the Selected Block</string>
      </property>
     </widget>
     <widget class="QPushButton" name="enterE">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>290</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Submit</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_3">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>220</y>
        <width>271</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Track Circuit Failure</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>590</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>540</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Set Temp F</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="heat_stat">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>420</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>110</y>
        <width>231</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Broken Rail</string>
      </property>
     </widget>
     <widget class="QPushButton" name="enterTemp">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>640</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Enter</string>
      </property>
     </widget>
     <zorder>textBrowser</zorder>
     <zorder>lineEdit</zorder>
     <zorder>enterB</zorder>
     <zorder>tableView</zorder>
     <zorder>lineEdit_s</zorder>
     <zorder>enterS</zorder>
     <zorder>tableView_S</zorder>
     <zorder>lineEdit_t</zorder>
     <zorder>enterT</zorder>
     <zorder>tableView_T</zorder>
     <zorder>refresh</zorder>
     <zorder>selectE</zorder>
     <zorder>checkBox_2</zorder>
     <zorder>label_5</zorder>
     <zorder>label_18</zorder>
     <zorder>enterH</zorder>
     <zorder>label_17</zorder>
     <zorder>enterE</zorder>
     <zorder>checkBox_3</zorder>
     <zorder>spinBox</zorder>
     <zorder>label</zorder>
     <zorder>heat_stat</zorder>
     <zorder>checkBox</zorder>
     <zorder>enterTemp</zorder>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Section View</string>
     </attribute>
     <widget class="QTableView" name="tableView_Sect">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>1111</width>
        <height>751</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Select Section</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="select_sect">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>0</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="enterSect">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>0</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Enter</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>All Stations</string>
     </attribute>
     <widget class="QTableView" name="tableView_Sall">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1121</width>
        <height>791</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1307</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
